src/tests/user.test.js

```diff
@@ -0,0 +1,59 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import Token from '../helpers/tokenHelper';
+import app from '../index';
+import mockData from './mockdata';
+
+const user = mockData.veryfyEmailUser;
+
+const token = Token.generateToken(user);
+chai.use(chaiHttp);
+chai.should();
+
+dotenv.config();
+
+describe('Authentication test', () => {
+  it('Test mock email route', (done) => {
+    chai.request(app).get('/').end((err, res) => {
+      res.should.have.status(200);
+      res.body.should.be.an('object');
+      done();
+    });
+  });
+  it('Test mock email route', (done) => {
+    chai.request(app).get('/api/users/mock-email').end((err, res) => {
```

Comment:
rename this file using camelCase, ex: userTest.js

Reply:
Thanks willy, I have changed the file name